invite/src/main/java/Main.java

```diff
@@ -11,6 +11,11 @@
 public class Main {
     static ArrayList<Invitee> invited = new ArrayList<>();
     static ArrayList<Invitee> notInvited = new ArrayList<>();
+
+    /**
+     *
```

Comment:
When I say add comment blocks I mean you to put stuff in them.  Please make sure to fill them out moving forward.